zap/src/main/java/org/zaproxy/zap/view/MultipleRegexesOptionsPanel.java

```diff
@@ -212,8 +226,13 @@ protected void init() {
         @Override
         protected boolean validateFields() {
             try {
-                Pattern.compile(getRegexTextField().getText(), Pattern.CASE_INSENSITIVE);
-            } catch (PatternSyntaxException e) {
+                String trimmedRegex = getRegexTextField().getText().trim();
+                if (trimmedRegex.trim().isEmpty()) return false; // to prevent accepting empty regex
```

Comment:
This is syntactically correct but it would be best to use curly braces for consistency

Reply:
Also why trim the trimmed value, is there a reason to apply it twice?

Reply:
to make sure that it's trimmed 😂
ok i'll fix that

Reply:
Better trim in `checkAndEnableConfirmButton()`, not worth enable the button if we are not going to accept it.

Reply:
@thc202 did this good suggestion